docs/en_US/nnicli_ref.md

```diff
@@ -0,0 +1,41 @@
+# NNI Client
+
+NNI client is a python API of `nnictl`, which implements the most common used commands. Users can use this API to control their experiments, collect experiment results and conduct advanced analyses based on experiment results in python code directly instead of using command line. Here is an example:
+
+```
+import nnicli as nc
+
+nc.start_experiment('nni/examples/trials/mnist-pytorch/config.yml', port=9090) # start an experiment
+
+nc.set_endpoint('http://localhost:9090') # set the experiment's endpoint, i.e., the url of Web UI
```

Comment:
what is this API used for? after an experiment is created, the endpoint is fixed, right? why we need to set it?

Reply:
i see, need to set this endpoint in order to query restful APIs.

Reply:
is it possible to automatically detect this endpoint? because it is counter intuitive that user is required to set endpoint

Reply:
Agree with you. Maybe we can get the endpoint when user use `start_experiment`. But if user wants to conduct some operation on other experiments, they still need to set the endpoint by their own.